### PR TITLE
chore(header): add alert into mocked data and update library

### PR DESCRIPTION
**Pull request by cbernat**
image

### CODE REVIEW

File: src/headerData.json [hunk 1]
```diff
@@ -175,5 +175,17 @@
...
       }
     ]
+  },
+  "alert":{
+    "type":"blocker",
+    "message":"Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.",
+    "button":{
+    "action":"validateSubscribersPopup",
+    "text":"Verificar Ahora"
+    },
```
Comment (andresmoschini) [inline src/headerData.json]: Indentation?

Comment (andresmoschini) [reply src/headerData.json]: Spaces?

File: src/headerData.json [hunk 2]
```diff
@@ -175,5 +175,17 @@
         "$$hashKey":"object:26"
       }
     ]
+  },
+  "alert":{
+    "type":"blocker",
+    "message":"Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.",
+    "button":{
+    "action":"validateSubscribersPopup",
```
Comment (andresmoschini) [inline src/headerData.json]: It is out of the scope of your task, but this kind of alert will not work in our webapp, right?

**Comment by andresmoschini** [comment]
image

File: src/headerData.json (hunk 2 above)
Comment (cbernat) [reply src/headerData.json]: Yes you are right. This button activates a popup.

**Comment by cbernat** [comment]
@andresmoschini yes it is broken. I will add a bug to fix this. There is an issue with the class that we use from webapp and the one that they use in UI library

File: src/headerData.json (hunk 1 above)
Comment (andresmoschini) [reply src/headerData.json]: Spaces where fine, deleting my previous comment...

**Pull request merged**